app/components/avo/modal_component.html.erb

```diff
@@ -3,19 +3,26 @@
   data-modal-target="modal"
 >
   <div aria-expanded="true" class="modal-overlay absolute w-full h-full bg-opacity-25 bg-gray-800 flex justify-center items-center" data-action="click->modal#close"></div>
-  <div aria-expanded="true" role="dialog" aria-modal="true" class="modal-body rounded-lg inset-auto w-11/12 lg:w-1/2 sm:max-w-168 min-h-1/4 bg-white flex z-50 relative shadow-modal overflow-auto max-h-full">
+  <div aria-expanded="true" role="dialog" aria-modal="true" class="modal-body rounded-lg inset-auto bg-white flex z-50 relative shadow-modal overflow-auto  <%= width_classes %> <%= height_classes %>">
     <div class="flex-1 flex flex-col justify-between">
       <div>
-        <div class="p-6 text-2xl tracking-normal font-semibold text-black">
-          <%= heading %>
-        </div>
-        <div class="px-6 text-base text-gray-500">
-          <%= content %>
-        </div>
-      </div>
-      <div class="flex justify-end items-baseline space-x-4 p-4 bg-gray-100">
-        <%= controls %>
+        <% if heading? %>
+          <div class="p-6 text-2xl tracking-normal font-semibold text-black">
+            <%= heading %>
+          </div>
+        <% end %>
+
+        <% if content? %>
```

Comment:
I think this `content?` call is causing a NoMethodError

Reply:
@segiddins  can you describe the scenario? I wasn't able to reproduce it

Reply:
https://github.com/rubygems/rubygems.org/pull/4154 yeah, tests started failing on the version bump

Reply:
👀

Reply:
I think you might need to update `view_component` on your end.
I can't put my finger on it exactly, but at some point they changed the way they figure out if the slot has any content inside.

Reply:
I see that view_component v3 added this helper. Are you able to update to VC 3?
I see that you are already on 2.8+ which should not bring breaking changes AFAIK.